roles/download/defaults/main.yml

```diff
@@ -8,6 +8,8 @@ calico_version: v0.19.0
 calico_cni_version: v1.2.1
 weave_version: v1.5.0
 
+cni_version: "v0.2.2"
```

Comment:
This doesn't have anything to do with fan, this is the release of the "standard" cni tools, so it's the release of https://github.com/containernetworking/cni.

Actual fan cni works by just utilizing the `bridge` plugin from that release.